src/utils/stringUtils.ts

```diff
@@ -15,7 +15,51 @@ class stringUtils {
      * @param replace String to replace keywords with.
      * @returns Replaced String.
      */
-    public static async customReplace(string: string, keyword: string, replace: string) {
+     public static async customReplace(string: string, keyword: string, replace: string) {
         return string.replace(new RegExp(keyword, "g"), replace);
     }
+
+    /**
+     * ### URL Parse
+     * @param string URL that want to be parsed
+     * @returns Dictionary
+     * {
+     *  URLData: origin URL result,
+        searchParams: Search parms, (To get parms, use .get("key") method)
+        spitted: Splitted URL (['https://','','polytoria.com'])
+     * }
+    */
+
+    public static async URLParse(string: string) {
+        let url = new URL(string);
+        let searchParams_url = url.searchParams
+        let spitted_url:Array<any> = string.split("/")
+
+        return {
+            URLData: url,
+            searchParams: searchParams_url,
+            spitted: spitted_url
+        }
+    }
+
+    /**
+     * ### Get Numbers From URL
+     * @param string URL that want to be parsed

```

Comment:
Use proper JSDoc formatting.
* @param `{ typeHere }` `nameOfTheVariable` `Description of the variable`
```js
/**
* @param {string} url URL to be inserted.
*